_includes/codepens/basic-conf/index.js

```diff
@@ -1,17 +1,18 @@
 tinymce.init({
   selector: '#basic-conf',
+  content_css: '//www.tiny.cloud/css/codepen.min.css',
   width: 600,
   height: 300,
   plugins: [
     'advlist autolink link image lists charmap print preview hr anchor pagebreak spellchecker',
     'searchreplace wordcount visualblocks visualchars code fullscreen insertdatetime media nonbreaking',
     'table emoticons template paste help'
   ],
-  toolbar: 'undo redo | styleselect | bold italic | alignleft aligncenter alignright alignjustify |' +
+  toolbar: 'undo redo | styleselect | bold italic | alignleft aligncenter alignright alignjustify | ' +
```

Comment:
This now has two spaces? That should be fine, but maybe we should remove the space on the next line just to be safe?

Reply:
fixed

Reply:
https://github.com/tinymce/tinymce-docs/pull/1426/commits/49f17aabc2e94afcff276b9e332dab05e39d67ae